lib/_http_incoming.js

```diff
@@ -47,7 +47,7 @@ function IncomingMessage(socket) {
   // response (client) only
   this.statusCode = null;
   this.statusMessage = null;
-  this._client = socket; // deprecated
+  this.client = socket;
```

Comment:
Oh, but then we will be creating many functions, whenever we create a new object. It might impact the performance. Not sure if it is okay.